### PR TITLE
IPython v7.10.0 + update requirements

### PR DESCRIPTION
**Pull request by Carreau**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Carreau** [comment]
How do I tell it it's not compatible with 3.6+ (dropped 3.5) ? linter is unhappy when I set a version number for Python.

**Comment by Carreau** [comment]
I guess it does not really matter as we don't build on 3.5 anyway...

File: recipe/meta.yaml [hunk 1]
```diff
@@ -30,11 +30,10 @@ requirements:
     - jedi >=0.10
     - pexpect  # [unix]
     - pickleshare
-    - prompt_toolkit >=2.0.0,<3
+    - prompt_toolkit >=2.0.0,<4
     - pygments
     - python
     - traitlets >=4.2
-    - win_unicode_console >=0.5  # [win and py<36]
```
Comment (bollwyvl) [inline recipe/meta.yaml]: did we somehow internalize this feature?

Comment (bollwyvl) [reply recipe/meta.yaml]: oh, no seeing the bit about not supporting 3.5...

Comment (Carreau) [reply recipe/meta.yaml]: No we dropped 3.5

Comment (bollwyvl) [reply recipe/meta.yaml]: add a skip up in build!

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

**Comment by Carreau** [comment]
> `skip: True  # [py<36]`

Ah Yes Thanks.

**Comment by bollwyvl** [comment]
LGTM if CI passes...

**Comment by Carreau** [comment]
> LGTM if CI passes...

❤️

**Pull request merged**